src/test/java/ca/uhn/fhir/jpa/starter/ExampleServerDstu2IT.java

```diff
@@ -1,5 +1,12 @@
 package ca.uhn.fhir.jpa.starter;
 
+import ca.uhn.fhir.context.FhirContext;
```

Comment:
I don't love this re-ordering of imports. Can these changes be removed from this commit?